src/content/getting-started/developers/react.md

```diff
@@ -43,11 +57,6 @@ $ yarn add carbon-components-react carbon-components carbon-icons
 
 3.  For older browsers (e.g. IE11), polyfills listed in the [carbon-components-react/.storybook/polyfills.js file](https://github.com/carbon-design-system/carbon-components-react/blob/master/.storybook/polyfills.js) are required.
 
-If you just want to try out `carbon-components-react`, you can also use [CodeSandbox](https://codesandbox.io).
```

Comment:
was this removed on purpose? there are a bunch of updates here that don't seem related to images?